mteb/tasks/Classification/multilingual/TurkicClassification.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+import datasets
+from datasets import DatasetDict
+from collections import Counter
+
+from mteb.abstasks import AbsTaskClassification, MultilingualTask
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class TurkicClassification(MultilingualTask, AbsTaskClassification):
```

Comment:
run `make lint` locally before pushing changes, that should fix the linting failing

Reply:
@dipam7 thanks, there were two issues: one was the setting bibtext to None and the other was it was trying to run tests for task that have been removed.